src/ReactiveUI.Validation/Contexts/ValidationContext.cs

```diff
@@ -102,7 +99,7 @@ public IObservable<bool> Valid
     /// <summary>
     /// Gets get the list of validations.
     /// </summary>
-    public ReadOnlyObservableCollection<IValidationComponent> Validations => _validations;
+    public IObservableList<IValidationComponent> Validations { get; }
```

Comment:
Why are we replacing a ROOC with `IObservableList` here? This is a breaking change, so a brief explanation of the reasoning behind this change could help. If this improves performance then worth showing the numbers. Thanks!